### PR TITLE
Modifications to L1REPACK:FullSimTP to prevent crashes and make it more compatible with other REPACK options

### PR DESCRIPTION
**Pull request by eyigitba**
#### PR description:

This PR started as a modification to L1REPACK:FullSimTP to prevent a crash due to missing GEM unpacker. However, while going through the config I realized some outdated options which I ended up removing. This config was already modifed to work only with L1T Stage-2 options, so there was no reason to keep around CSCTF/DTTF options.

Furthermore, I am changing the config for now to use unpacked muon TF inputs instead of calling muon TP re-emulation modules. This is mostly done to bypass a problem I observe in DT TP re-emulation that causes lower rates. I kept the muon TP re-emulation options in place to allow people to switch quickly between the two options. 

#### PR validation:

I validated by running L1REPACK:uGT and L1REPACK:FullSimTP and comparing them. There are no crashes and the final rates are comparable up to re-emulation level mismatches. 

The normal CMSSW workflows are probably no enough to check this PR, otherwise the crash would have been caught before. 

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

I think we will backport this to 14_0_X in case we need to use it again in this year's data workflows. It's not urgent, so I would wait until the discussion on this PR is settled.

FYI @missirol

### CODE REVIEW

File: Configuration/StandardSequences/python/SimL1EmulatorRepack_FullSimTP_cff.py [hunk 1]
```diff
@@ -114,7 +140,7 @@
 
 
 SimL1EmulatorTask = cms.Task()
-stage2L1Trigger.toReplaceWith(SimL1EmulatorTask, cms.Task(unpackEcal,unpackHcal,unpackCSC,unpackDT,unpackRPC,unpackEmtf,unpackCsctf,unpackBmtf
+stage2L1Trigger.toReplaceWith(SimL1EmulatorTask, cms.Task(unpackEcal,unpackHcal,unpackCSC,unpackDT,unpackRPC,unpackGEM,unpackEmtf,unpackBmtf,unpackOmtf
```
Comment (missirol) [inline Configuration/StandardSequences/python/SimL1EmulatorRepack_FullSimTP_cff.py:143]: Small point: iiuc, GEM has its own modifier, so maybe something like the following should be done.
https://github.com/cms-sw/cmssw/blob/1e7c61e8e118871e99e5aa53bd6b396101f5eb75/Configuration/StandardSequences/python/SimL1EmulatorRepack_Full_cff.py#L157-L165

**Comment by missirol** [comment]
@eyigitba 

Thanks for this update ! Indeed, I think it would be useful to backport it to 14_0_X.

**Comment by eyigitba** [comment]
Thanks @missirol ! I'll make the backport now.

**Comment by Martin-Grunewald** [comment]
please test

**Comment by antoniovilela** [comment]
+1

**Pull request merged**